### PR TITLE
refactor(theme)!: copy composable helpers to theme

### PR DESCRIPTION
**Pull request by ghost**
## Description

Those helpers are only used in composables so they're only ever useful for people who will be porting helpers. There are no paths to change in the theme
## Related Issue
https://vsf.atlassian.net/browse/M2-282
https://vsf.atlassian.net/browse/M2-283
https://vsf.atlassian.net/browse/M2-284

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Review by RetroModernDev — CHANGES_REQUESTED** [review]
Ca you also update all places where those helpers are used?

**Comment by sethidden** [comment]
@Frodigo As I said in the description - they are not used anywhere. They're only used in composables that we haven't ported yet

File: packages/theme/helpers/userAddressManipulator.ts [hunk 1]
```diff
@@ -0,0 +1,78 @@
+import { CustomerAddressInput } from '@vue-storefront/magento-api';
```
Comment (bartoszherba) [inline packages/theme/helpers/userAddressManipulator.ts:1]: Can you get rid of this dependency?

File: packages/theme/helpers/htmlDecoder.ts [hunk 2]
```diff
@@ -0,0 +1,13 @@
+export function htmlDecode(input: string) {
```
Comment (bartoszherba) [inline packages/theme/helpers/htmlDecoder.ts:1]: This helper is used only in buildCategoryTree helper only in composable package, therefore we do not need it in the theme.

**Review by bartoszherba — CHANGES_REQUESTED** [review]

File: packages/theme/helpers/userAddressManipulator.ts (hunk 1 above)
Comment (sethidden) [reply packages/theme/helpers/userAddressManipulator.ts]: https://github.com/vuestorefront/magento2/pull/734/commits/86147ed386a74965d9b986b529e0ac44224a3f58

File: packages/theme/helpers/htmlDecoder.ts (hunk 2 above)
Comment (sethidden) [reply packages/theme/helpers/htmlDecoder.ts]: Done

**Review by RetroModernDev — DISMISSED** [review]

**Review by bartoszherba — APPROVED** [review]

**Review by RetroModernDev — APPROVED** [review]

**Pull request merged**